ehelgueroc/mern-apollo
# Configured vitest with rtl on client and created first test

## Problem
Need to test the components created on the project.

## Solution
- Configured Vitest
- Configured React Testing Library

## Testing steps
- Execute the tests on the client folder
- All tests should be green
- - Change some of them to know that the configuration is working as expected


## Additional information
Note: With version 4.1.0 of Vitest, the vite.config.ts was giving errors trying to add the test key on the configuration object. I had to upgrade it to version 4.2.1

----

## For reviewers

This section is for reviewers of this PR.

**At least one approver of this PR should complete below checklist.**

- [x] I understand what this PR is doing. Its purpose and the solution.
- [ ] I am happy with the description of this PR. I believe it sufficiently describes the problem being solved, has good testing steps and additional information (including updated documentation where appropriate).
- [ ] I have concerns about this PR but I am still approving it. I have expressed my concerns in the comments of this PR.
